### DISCUSSION TITLE
Filament View - A Filament and Flutter integration

### DISCUSSION THREAD

**Discussion by jwinarske** [google/filament#8160] [General]
@romainguy update on our (Toyota Connected North America) end.  We have latest Filament integrated with latest Flutter version.  We have multiple areas we are augmenting in the filament source tree:
1. gltfio individual model loading done callback - speeds up load time
2. RenderableManager::Builder finalizer - required to prevent leaks 
3. Camera manipulator inertia settings - usability
4. CMake generic installation of libraries - basic requirement
5. Hit testing (in single frame)

More in the coming months.  I think these changes would benefit all users of Filament.  Let me know if you're up for a call to discuss.

![370030047-d1250332-4a7d-4944-9990-df85ee10e9f1](https://github.com/user-attachments/assets/d5268609-7dd5-42fc-985e-8c6204b22b36)

**Comment by romainguy** [comment]
1-3 seem like no-brainers, 5 probably as well although I'm sure @pixelflinger would love to know how it's done without stalling the pipeline (are you testing against geometry instead of using the GPU pipeline?). 4 also sounds great since it's been requested by other folks.

**Reply by pixelflinger** [reply]
Yes all sounds reasonable. I'm happy to do a call.

**Reply by jwinarske** [reply]
@pixelflinger I'll send out an invite for something in the week of October 14th.  I'm out starting next week.
